libPhoneNumberTests/NBPhoneNumberUtilTest.m

```diff
@@ -1947,6 +1948,26 @@ - (void)testParseNationalNumberArgentina {
   XCTAssertTrue([arNumber isEqual:[_aUtil parse:@"023 1234 0000" defaultRegion:@"AR" error:&anError]]);
 }
 
+- (void)testParseNationalNumberRussia {
+    NSError *anError = nil;
+    // Test parsing mobile numbers for Russia.
+    NBPhoneNumber *ruNumber = [[NBPhoneNumber alloc] init];
+    [ruNumber setCountryCode:@7];
+    [ruNumber setNationalNumber:@8125555646];
+    
+    XCTAssertTrue([ruNumber isEqual:[_aUtil parse:@"+7 812 555 56-46" defaultRegion:@"RU" error:&anError]]);
```

Comment:
XCTAssertEqual(ruNumber, [_aUtil parse:@"+7 812 555 56-46" defaultRegion:@"RU" error:&anError], @"Error: %@", anError)

Reply:
I assumed it should be same as in other test methods, where mostly XCTAssertTrue is used. But, now I changed it to XCTAssertEqualObjects.